src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/Rules/RazorGenerateWithTargetPath.cs

```diff
@@ -10,58 +10,58 @@
 
 namespace Microsoft.CodeAnalysis.Razor.ProjectSystem.Rules
```

Comment:
Generated file, automatic change. Committing in so it doesn't show up on future diffs.